lib/gpu/render_pass.cc

```diff
@@ -241,13 +241,24 @@ Dart_Handle InternalFlutterGpu_RenderPass_SetDepthStencilAttachment(
     int stencil_load_action,
     int stencil_store_action,
     int stencil_clear_value,
-    flutter::gpu::Texture* texture) {
+    flutter::gpu::Texture* texture,
+    Dart_Handle resolve_texture_wrapper) {
+  flutter::gpu::Texture* resolve_texture = nullptr;
+
+  if (!Dart_IsNull(resolve_texture_wrapper)) {
+    resolve_texture = tonic::DartConverter<flutter::gpu::Texture*>::FromDart(
+        resolve_texture_wrapper);
+  }
+
   {
     impeller::DepthAttachment desc;
     desc.load_action = flutter::gpu::ToImpellerLoadAction(depth_load_action);
     desc.store_action = flutter::gpu::ToImpellerStoreAction(depth_store_action);
     desc.clear_depth = depth_clear_value;
     desc.texture = texture->GetTexture();
+    if (resolve_texture) {
```

Comment:
I don't think depth/stencil attachments should have a resolve field.

Reply:
maybe I made a mistake when reading this but it looks like this was from the same attachment as the color attachment but maybe that isn't the case.

Feel free to open the can of worms as needed